### ISSUE TITLE
Re-ordering breaks category index?

### ISSUE THREAD

**Issue by husseinalhammad** [Pipits/pipit-catalog#3]
Tony Astley [reported](https://community.perchcms.com/forum/thread/1319-perch-shop-show-only-products-from-top-level-of-current-category-not-sub-categor/?postID=5214#post5214) that filtering by `_category` does not work after re-ordering the products until the products are manually saved.

> Any ideas why re-ordering the products using pipit catalog would initially break the above filter - following re-order the products disappear, and only show again once the product is saved?

```php
perch_shop_products([ 
	'filter' => '_category',
	'match' => 'eq',
	'value' => $fullcatPath,
]); 
```

**Comment by husseinalhammad** [comment]
The only way I can replicate this is when the category field is not present in the product edit form template and hence `PerchAPI_Base::index()` removing the `_category` rows from the index table.

**Comment by husseinalhammad** [comment]
Turns out the issue is that pipit_catalog does not use the correct default perch_shop product template when a user template is not present. No template = no index!

**Comment by husseinalhammad** [comment]
This has been addressed in the [v1.x](https://github.com/Pipits/pipit-catalog/tree/v1.x) branch and the fix will be available in the next release.

**Comment by husseinalhammad** [comment]
Fixed in v1.4
